Libraries/Text/TextInput/Singleline/RCTUITextField.m

```diff
@@ -65,7 +65,7 @@ - (NSText *)setUpFieldEditorAttributes:(NSText *)textObj
   NSMutableDictionary *selectTextAttributes = fieldEditor.selectedTextAttributes.mutableCopy;
   selectTextAttributes[NSBackgroundColorAttributeName] = self.selectionColor ?: [NSColor selectedControlColor];
 	fieldEditor.selectedTextAttributes = selectTextAttributes;
-  fieldEditor.insertionPointColor = self.selectionColor ?: [NSColor selectedControlColor];
+  fieldEditor.insertionPointColor = self.selectionColor ?: [RCTUIColor selectedControlColor];
```

Comment:
Missing the TODO comment

Reply:
This change is already inside a ` // [TODO(macOS ISS#2323203)` ... ` // ]TODO(macOS ISS#2323203)` block.